### DISCUSSION TITLE
REGAIN COMMUNITY TRUST

### DISCUSSION THREAD

**Discussion by maallam24** [starknet-io/starknet-docs#1256] [Ideas]
-Is your feature request related to a problem? Please describe.

   If you've been on CT since the airdrop you have seen the constant FUD that is staining the starknet name which clearly is on a downward spiral. We must prioritize community it has become abundantly clear that without community in web3 projects will always fail. Consider Starknet is the most technological advanced chain in web3 so how come people arent using it? Quite simple the community felt betrayed after the first airdrop which caused a feedback loop and now any tweet even mentioning starknet gets witch hunted.

-Describe the solution you'd like

   The first priority needs to be how can we earn the communities trust back? This may seem like the most difficult step and it is, however once trust is reestablished the rest will all come together. As stated before Stark has the best tech of all the chains , people want to use it but they will likely never use it if the first airdrop is not fixed. Regardless of future airdrop rounds in the future this is not enough incentive for someone to return to a chain whom they believe betrayed them. My solution , reward users who were only not eligible due to the lack of .005 eth (the amount of $strk here does not matter) people will return to the chain as long as they feel they were not betrayed and the team is taking some steps to remedy the situation.

-Describe alternatives you've considered

   alternative solutions will be difficult because as long as the people from the first round are not rewarded with something, the stain on Starknet will remain. W e can assume most who were not eligible due to the .005 eth criteria are not plenty and then the others who were not eligible regardless of the .005 eth criteria. Secondary solution reward the users who would have been eligible if not for the .005 eth with the unclaimed $STRK tokens from the first airdrop.

-Additional context

    I have been in web3 for a long time and i know how this story ends, without community it is nearly impossible to succeed. Seeing a chain with superior tech like Starknet getting railroaded because of a minor mistake is a tough pill to swallow but by no means is it not fixable.

**Comment by stoobie** [comment]
@maallam24 This is a duplicate of #1255 . Please see the response thread there, and thank you for reaching out.

I'm closing this discussion, as this is not the correct place for it.
